src/components/calculator.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+function Calculator() {
+  return (
+    <section>
```

Comment:
- [ ] Kindly make the Calculator component have at least 1 child component as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Create%20at%20least%201%20child%20component).